dap_prinz_green_jobs/config/extract_green_skills_esco.yaml

```diff
@@ -0,0 +1,20 @@
+ner_model_path: "outputs/models/ner_model/20220825/"
+taxonomy_name: "green_esco"
+taxonomy_path: "outputs/data/skill_ner_mapping/green_esco_data_formatted.csv"
+clean_job_ads: True
+min_multiskill_length: 75
+taxonomy_embedding_file_name: "outputs/data/skill_ner_mapping/green_esco_embeddings.json"
+prev_skill_matches_file_name: ""
+hard_labelled_skills_file_name: ""
+hier_name_mapper_file_name: ""
+num_hier_levels: 0
+skill_type_dict:
+  { "skill_types": ["preferredLabel", "altLabels"], "hier_types": [] }
+match_thresholds_dict:
+  { "skill_match_thresh": 0.7 }
```

Comment:
In a future PR we should experiment with this threshold and see which value gives us a good precision in the matches